components/subscriptions-v2/SubscriptionsContainer.js

```diff
@@ -0,0 +1,68 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+import { Flex } from '../Grid';
+
+import StyledSubscriptionsCard from './StyledSubscriptionsCard';
+
+const AllCardsContainer = styled(Flex).attrs({
+  flexWrap: 'wrap',
+  justifyContent: 'space-evenly',
+})``;
+
+const CollectiveCardContainer = styled.div`
+  width: 280px;
+  padding: 20px 15px;
+`;
+
+const SubscriptionsContainer = ({ subscriptions, filter, createNotification }) => {
+  const [isHovering, setHovering] = useState(null);
+
+  const activeSubscriptions = subscriptions.nodes.filter(sub => sub.status === 'ACTIVE');
+  const monthlySubscriptions = activeSubscriptions.filter(sub => sub.frequency === 'MONTHLY');
+  const yearlySubscriptions = activeSubscriptions.filter(sub => sub.frequency === 'YEARLY');
+  const cancelledSubscriptions = subscriptions.nodes.filter(sub => sub.status === 'CANCELLED');
```

Comment:
When filtering the Monthly, do we actually want the Monthly Active? Same for yearly, I think this is what would make the most sense for me.

Reply:
True! Added